package-lock.json

```diff
@@ -17740,16 +17740,16 @@
       }
     },
     "node_modules/typescript": {
-      "version": "5.6.2",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-5.6.2.tgz",
-      "integrity": "sha512-NW8ByodCSNCwZeghjN3o+JX5OFH0Ojg6sadjEKY4huZ52TqbJTJnDo5+Tw98lSy63NZvi4n+ez5m2u5d4PkZyw==",
+      "version": "4.9.5",
+      "resolved": "https://registry.npmjs.org/typescript/-/typescript-4.9.5.tgz",
+      "integrity": "sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==",
```

Comment:
@DionisioRD is it a typo?

Reply:
It is an auto-generated file so no changes are done manually in the package-lock

Reply:
Not sure why it downgrade a package :eyes:

Reply:
Maybe, a way to ensure nothing is wrong is to remove the file and run `npm i` again, wdyt?

Reply:
Done :+1: it's a completely new file now.